modules/cluster-logging-collector-tuning.adoc

```diff
@@ -0,0 +1,178 @@
+
+By default in {product-title}, Fluentd uses the _exponential backoff_ retry method, where Fluentd doubles the time it waits between attempts to retry flushing, which helps reduce re-connect requests with the destination. You can disable exponential backoff and use the _periodic_ retry method instead, which retries writing off the chunks at a specified interval. 
+
+By default, Fluentd retries flushing the chunks indefinitely. You can change this behavior and discard chunks that cannot be properly flushed.
+
+You configure chunking and flushing behavior using the following parameters in the Cluster Logging Custom Resource (CR). The parameters are then automatically added to the Fluentd ConfigMap.
+
+[NOTE]
+====
+You can use these parameters to help determine the trade-offs between latency and throughput. If you want to optimize cluster logging for throughput, you could batch to reduce network packet count; configure larger buffers and queues, delay flushes, and set longer times between retries. To optimize for low latency, you could send data as soon as possible, avoid build-up of batches, have shorter queues and buffers, and use more frequent flush and retries.
+====
+
+These settings are:
+
+* not relevant to most users, default settings should give good general performance
+* only for advanced users with detailed knowledge of Fluentd configuration and performance
+* only for performance tuning, they have no effect on functional aspects of logging
+
+.Advanced Fluentd Configuration Parameters
+[options="header"]
+|===
+
+|Parmeter |Description |Default
+
+|`chunkLimitSize`
+|The maximum size of each chunk. Fluentd stops writing data in a chunk when it reaches this size.
+|`8m`
+
+|`flushAtShutdown`
+|Determine whether to flush all chunks on shutdown. Leave blank for `false`.
+|`true`
+
+|`flushInterval`
+|The interval between chunk flushes. You can use "s" (seconds), "m" (minutes), or "h" (hours).
+|`1s`
+
+|`flushMode`
+a| The method to perform flushes: 
+* `lazy`: flushes chunks based on the `timekey` parameter.
+* `interval`: flushes chunks based on the `flushInterval`.
+* `immediate`: flushes chunks immediately after data is added to a chunk.
+|`interval`
+
+|`flushThreadCount`
+|The number of threads that perform chunk flushing. Increasing the number of threads improves the flush throughput to hide network latency.
+|`2`
+
+|`overflowAction`
+a|The chunking behavior when the queue is full:
+* `throw_exception`: raises an exception to show in the log.
+* `block`: stops data chunking until buffer full issue is resolved.
+* `drop_oldest_chunk`: drop the oldest chunk to accept new incoming chunks.
```

Comment:
A note here that users can use `drop_oldest_chunk` for log lines that work like metrics, where older values have less of value compared to newer.